web-client/src/presenter/computeds/formattedTrialSessionDetails.js

```diff
@@ -24,6 +24,13 @@ export const formattedTrialSessionDetails = (get, applicationContext) => {
         .formatPhoneNumber(formattedTrialSession.chambersPhoneNumber);
     }
 
+    formattedTrialSession.isHybridSession =
+      formattedTrialSession.sessionType === 'Hybrid';
```

Comment:
🔧 Could the entity constant `SESSION_TYPES` be used here instead of hardcoding?

Reply:
Good catch...I thought I had used EntityConstants every where I could.  Thanks.